### PR TITLE
[OPENTOK-43733] Log when RTT is really high

### PR DESCRIPTION
**Pull request by ggoldens**
#### What is this PR doing?
This PR adds a watcher to the publisher so that we can detect when we have really high values of rtt (=> 10000) either for video or audio.

#### How should this be manually tested?
Will be hard to test since we can't reproduce the scenario. But, we can make a change to see how it will log:

- Change the line 32 in the file `publisher-rtt-watcher.js` to this:
```
if (roundTripTime <= RTT_THRESHOLD) {
```
- npm run build
- npm start
- Open localhost:3000 and join a room
- go to network tab and check that every 30s you should see a new ClientEvent log with a data similar to this:

```
clientSystemTime: 1629925770740
clientVersion: "v2.20.3"
componentId: "meet.tokbox.com"
connectionId: "90c76575-2007-4d5b-b4b7-b6cd533747c1"
guid: "2c8e322b-0feb-4b6d-859d-2bed9ea98dbb"
kind: "video"
logVersion: "2"
name: "meet.tokbox.com"
partnerId: "100"
rtt: 0.218
sessionId: "1_MX4xMDB-fjE2Mjk5MjU1OTA5MzF-YUJpN1JHTTVnUC95TjRTMWtvRHZnVDlDfn4"
source: "http://localhost:3000/dfdsffds?tokenRole=moderator"
userAgent: "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.159 Safari/537.36"
```

Because of this change to test it, you will see two logs, one for audio and one for video. In real like, it will only log the video kind that has the high value. In case both have high values, it will log both (audio and video).

#### What are the relevant tickets?
OPENTOK-43733

### CODE REVIEW

**Review by albertoacn — DISMISSED** [review]
Reviewed and tested. LGTM 🚀

File: src/js/helpers/publisher-rtt-watcher.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+const start = (publisher, session) => {
+  stop();
+
+  interval = setInterval(() => {
+    // init analytics
+    if (session.connection.connectionId && !analytics.isInitiated()) {
+      analytics.init(session);
+    }
+
+    publisher.getRtcStatsReport().then((stats) => {
```
Comment (v-kpheng) [inline src/js/helpers/publisher-rtt-watcher.js:27]: The high values we're seeing might be due to a bug in https://github.com/opentok/webrtc-js/blob/develop/src/js/ot/peer_connection/qos/parseQOS.js.  Do you think it's worthwhile to add logging there instead or in addition to, @ggoldens ?

Please advise.  Thanks!

Comment (ggoldens) [reply src/js/helpers/publisher-rtt-watcher.js:27]: Yeap, we can include some logging there as well, but not 100% sure which logs you expect to see. Could you please provide a few examples? Thanks!

**Review by v-kpheng — APPROVED** [review]
LGTM! 🚀 💪

**Comment by michaelwhittemore** [comment]
I'm a little confused by what I'm seeing. I've made the change so that we log when the RTT time is *less* than the threshold and in the network tab every 30 seconds I see 3 logs for video and 1 for audio. Is this expected?

**Review by michaelwhittemore — APPROVED** [review]
Ah I think that the three video reports are from the three simulcast streams. In this case, LGTM

**Pull request merged**